server.go

```diff
@@ -9,24 +9,17 @@ import (
 
 // ServiceRegistry accumulates service definitions. Servers typically have this
 // interface for accumulating the services they expose.
-type ServiceRegistry interface {
-	// RegisterService registers the given handler to be used for the given
-	// service. Only a single handler can be registered for a given service. And
-	// services are identified by their fully-qualified name (e.g.
-	// "package.name.Service"). Attempting to register the same service more
-	// than once is an error that can panic.
-	RegisterService(desc *grpc.ServiceDesc, srv interface{})
-}
-
-var _ ServiceRegistry = (*grpc.Server)(nil)
+//
+// Deprecated: Use grpc.ServiceRegistrar instead.
+type ServiceRegistry = grpc.ServiceRegistrar
```

Comment:
You must have gotten the GRPC folks to lane this, I imagine?

Reply:
I had mentioned it in a few issues and was quite happily surprised to see that it had been done.